src/PDFs/physics/Amp4Body_TD.cu

```diff
@@ -401,16 +406,16 @@ __host__ Amp4Body_TD::Amp4Body_TD(std::string n,
     initialize();
 
     Integrator   = new NormIntegrator_TD();
-    redoIntegral = new bool[components.size() - 1];
-    cachedMasses = new fptype[components.size() - 1];
-    cachedWidths = new fptype[components.size() - 1];
+    redoIntegral = new bool[LineShapes.size()];
+    cachedMasses = new fptype[LineShapes.size()];
+    cachedWidths = new fptype[LineShapes.size()];
 
     for(int i = 0; i < LineShapes.size(); i++) {
         lscalculators.push_back(new LSCalculator_TD());
     }
 
-    // for (int i = 0; i < amp_idx.size(); i++)
-    //    printf("%i - %i\n", i, amp_idx[i]);
+    for(int i = 0; i < amp_idx.size(); i++)
+        printf("%i - %i\n", i, amp_idx[i]);
```

Comment:
These printouts should be protected by an if debug line.

Reply:
Thanks for finding those, those were not supposed to be committed.